src/components/Modals/AddContactModal.jsx

```diff
@@ -134,12 +81,7 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
       aria-labelledby="dialog-title"
```

Comment:
this `aria` isn't linked to anything. Nothing with this id.   It will be necessary when we use `<DialogTitle />` and `<DialogContent />` after removing `FormSection`